data/newtabicons-content.js

```diff
@@ -140,6 +153,6 @@ function overrideToggle() {
     updateThumbnails();
   });
 }
-
-addThumbnails(window.document.getElementsByClassName('newtab-cell'));
+const selectors = '.newtab-cell, .spotlight-image, .tile-img-container .site-icon-background'
```

Comment:
So, I know const variables get hoisted to the top of the file, but let's put this at the top of the file anyways so that people who hit line 28 don't wonder what `selectors` is…  😉